oe_theme.theme

```diff
@@ -2106,7 +2106,7 @@ function oe_theme_preprocess_paragraph__oe_carousel(array &$variables): void {
       'centered' => FALSE,
       // The paragraph has a required image field, so we need to set a default
       // image variant in order to render the image.
-      'variant' => 'image-gradient',
+      'variant' => 'text-highlight',
```

Comment:
As I understand we don't have any more 'gradient' variant for banner component but is it confirmed replacing to highlight variant (why not overlay for instance)?

Reply:
I replied in the ticket.